### DISCUSSION TITLE
Displaying REST query value in Stat Card

### DISCUSSION THREAD

**Discussion by Kuuchuu** [Budibase/budibase#3763] [Help] [answered]
I need to display the contents of a REST query in a stat card but I'm having no luck. The REST query returns only a single value after a JavaScript transformer. Here is my transformer:
```
var obj = data.results;
var sum = 0;
obj.forEach(function(item) {
    sum += parseFloat(item.ItemNetSales)
});

return sum.toFixed(2)
```
which will output this:
```
{
  "value": "125.02"
}
```
I see how I can get the stat card to show a data source's "Rows". But selecting that displays '[{"value":"125.02"}]', not just the value, "125.02". My attempts thus far to use a JavaScript function to display only the "125.02" has so far been unsuccessful. Any guidance would be greatly appreciated. Thanks!

**Comment by mjashanks** [comment, marked as answer]
So, all queries return arrays. If they don't, they get put inside an array. This is so that `Data Providers` have a consistent definition to work from.

So, what you need to do is:
```
> Data Provider : Data = Your Query
    > Repeater 
        > Stat Card
```

You could also use a `Repeater Block`...
```
> Repeater block : Data = Your Query
    > Stat Card
```
